### DISCUSSION TITLE
Add Timezone dropdown to Craft 3 install screen

### DISCUSSION THREAD

**Discussion by putyourlightson** [craftcms/cms#8502] [Ideas]
### Description

The Timezone dropdown would be very handy to have on the install screen in Craft 3, given that many people are not in PST.

Apologies if this has already been addressed.

**Comment by brandonkelly** [comment]
> given that many people are not in PST.

Isn’t UTC the default?

**Comment by putyourlightson** [comment]
Hmm, yes it appears to be, not sure why it defaulted to PST in my local install, I'll check my server settings.

**Comment by brandonkelly** [comment]
Cool thanks for clarifying. Maybe you have the `timezone` config setting set or something.

Not that it invalidates the feature request of course :)
